src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -40,7 +40,7 @@ public ResponseEntity<AdsDTO> getAds(){
                 @ApiResponse(responseCode = "401", description = "Unauthorized")}
         )
         @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-        public ResponseEntity<AdDTO> addAd(@RequestBody AdDTO ad) {
+        public ResponseEntity<AdDTO> addAd(@RequestParam AdDTO ad, @RequestPart MultipartFile image) {
```

Comment:
```suggestion
        public ResponseEntity<AdDTO> addAd(@RequestParam AdDTO properties, @RequestPart MultipartFile image) {
```
название части с JSON такое должно быть, иначе работать фронт не будет с таким эндпоинтом